source/jormungandr/tests/bragi_autocomplete_tests.py

```diff
@@ -118,8 +124,13 @@ def test_autocomplete_call_override(self):
         """"
         test that the _autocomplete param switch the right autocomplete service
         """
+        url = 'https://host_of_bragi/autocomplete'
+        kwargs = {'params': {u'q': u'bob', u'type[]': [u"['stop_area', 'address', 'poi', 'administrative_region']"],
```

Comment:
It must be isofunctional, thus:
 - by default, type[]=stop_area&type[]=address&type[]=poi&type[]=administrative_region
 - the type[] must be checked in {stop_area, stop_point, address, poi, administrative_region}
 - a type "conversion" may be needed to request bragi
 - for stop points, 2 possibilities:
    - or we silently remove stop point in the request to bragi
    - or we error with unsupported object